RealmSwift-swift2.2/Realm.swift

```diff
@@ -98,6 +98,9 @@ public final class Realm {
     /**
      Performs actions contained within the given block inside a write transaction.
 
+     If the block throws an exception, the transaction will be canceled, reverting any writes made in the block before
```

Comment:
"If the block throws an error". Apple never refers to Swift errors as "exceptions", but rather "Swift errors"